@@ Apply this patch to Hydra's API description in swagger 2.0 yaml format
@@ to fix some validation issues. Original yaml file was created by convertion
@@ of apib (API blueprint) Hydra's file from apiary with apimatic convertion API.

--- hydra.orig.yaml	2016-12-08 15:47:59.000000000 +0700
+++ hydra.fixed.yaml	2016-12-08 15:52:15.000000000 +0700
@@ -1,6 +1,6 @@
-swagger: 2.0
+swagger: "2.0"
 info:
-  version: 1.0
+  version: "1.0"
   title: ory-am/hydra
   description: 'TODO: Add a description'
   license:
@@ -26,7 +26,7 @@
       parameters: []
       responses:
         200:
-          description: 
+          description: "-"
       x-unitTests:
       - request:
           method: GET
@@ -58,7 +58,7 @@
       parameters: []
       responses:
         200:
-          description: 
+          description: "-"
       x-unitTests:
       - request:
           method: POST
@@ -104,13 +104,13 @@
           type: string
       responses:
         200:
-          description: 
+          description: "-"
           schema:
             $ref: '#/definitions/Oauth2TokenIntrospectionResponse'
         401:
-          description: 
+          description: "-"
         403:
-          description: 
+          description: "-"
       x-unitTests:
       - request:
           method: POST
@@ -154,7 +154,7 @@
       parameters: []
       responses:
         200:
-          description: 
+          description: "-"
           schema:
             $ref: '#/definitions/Oauth2ClientBase'
         400:
@@ -201,7 +201,7 @@
         description: The client's `id`.
       responses:
         200:
-          description: 
+          description: "-"
           schema:
             $ref: '#/definitions/Oauth2ClientBase'
         400:
@@ -233,7 +233,7 @@
         description: The client's `id`.
       responses:
         200:
-          description: 
+          description: "-"
         400:
           description: Unexpected error in API call. See HTTP response body for details.
         401:
@@ -262,7 +262,7 @@
         description: the set name, e.g. "app-tls-keys".
       responses:
         200:
-          description: 
+          description: "-"
           schema:
             $ref: '#/definitions/JsonWebKeySet'
       x-unitTests: []
@@ -286,7 +286,7 @@
         description: the set name, e.g. "app-tls-keys".
       responses:
         200:
-          description: 
+          description: "-"
           schema:
             $ref: '#/definitions/JsonWebKeySet'
       x-unitTests: []
@@ -315,7 +315,7 @@
           $ref: '#/definitions/GenerateANewJsonWebKeySetRequest'
       responses:
         200:
-          description: 
+          description: "-"
           schema:
             $ref: '#/definitions/JsonWebKeySet'
       x-unitTests: []
@@ -339,7 +339,7 @@
         description: the set name, e.g. "app-tls-keys".
       responses:
         200:
-          description: 
+          description: "-"
       x-unitTests: []
       x-operation-settings:
         CollectParameters: false
@@ -367,7 +367,7 @@
         description: "the set name, e.g. `shared´."
       responses:
         200:
-          description: 
+          description: "-"
           schema:
             $ref: '#/definitions/JsonWebKeySet'
       x-unitTests: []
@@ -396,7 +396,7 @@
         description: "the set name, e.g. `shared´."
       responses:
         200:
-          description: 
+          description: "-"
           schema:
             $ref: '#/definitions/JsonWebKeySet'
       x-unitTests: []
@@ -425,7 +425,7 @@
         description: "the set name, e.g. `shared´."
       responses:
         200:
-          description: 
+          description: "-"
       x-unitTests: []
       x-operation-settings:
         CollectParameters: false
@@ -448,7 +448,7 @@
         description: only return relevant policies for `subject`, e.g. `max`.
       responses:
         200:
-          description: 
+          description: "-"
           schema:
             $ref: '#/definitions/PolicyBase'
       x-unitTests: []
@@ -467,7 +467,7 @@
       parameters: []
       responses:
         200:
-          description: 
+          description: "-"
           schema:
             $ref: '#/definitions/PolicyBase'
       x-unitTests:
@@ -507,7 +507,7 @@
         description: the policy's id.
       responses:
         200:
-          description: 
+          description: "-"
           schema:
             $ref: '#/definitions/PolicyBase'
       x-unitTests: []
@@ -531,7 +531,7 @@
         description: the policy's id.
       responses:
         200:
-          description: 
+          description: "-"
       x-unitTests: []
       x-operation-settings:
         CollectParameters: false
@@ -554,7 +554,7 @@
           $ref: '#/definitions/CheckIfTheSubjectOfATokenIsAllowedToDoSomethingRequest'
       responses:
         200:
-          description: 
+          description: "-"
           schema:
             $ref: '#/definitions/CheckIfTheSubjectOfATokenIsAllowedToDoSomethingResponse'
         401:
@@ -602,7 +602,7 @@
           $ref: '#/definitions/CheckIfASubjectIsAllowedToDoSomethingRequest'
       responses:
         200:
-          description: 
+          description: "-"
           schema:
             $ref: '#/definitions/CheckIfASubjectIsAllowedToDoSomethingResponse'
         401:
@@ -685,16 +685,6 @@
         type: array
         items:
           type: string
-  SsoConnectionBase:
-    title: SSO Connection Base
-    type: object
-    properties:
-      provider:
-        type: string
-      localSubject:
-        type: string
-      remoteSubject:
-        type: string
   Oauth2ClientBase:
     title: OAuth2 Client Base
     type: object
